packages/tokens/README.md

```diff
@@ -41,3 +37,18 @@ import { spacing } from '@leafygreen-ui/tokens';
 
 const example = () => <span style={{ margin: spacing[1] }}>Hello World</span>;
 ```
+
+## Fonts
+
+```js
+import { fonts } from '@leafygreen-ui/tokens';
+
+/**
+ * const fonts = {
```

Comment:
We should...not do this, cuz then it's another place we need to keep maintained as we change things. 

Right now spacing is already out of date.

Better to just link to the source code directly IMO.

Reply:
👍